src/app/AnswerVeryfication.js

```diff
@@ -0,0 +1,3 @@
+export function isAnswerCorrect(correctAnswer, playerAnswer) {
+    return correctAnswer == playerAnswer;
+}
```

Comment:
Jeśli to ma być tak prosty kod to jest git!

Reply:
Jest git :) Jakbyśmy dodawali komputer rozpoznający obrazy - wtedy trzeba bardziej pokombinować. Ten losujący z możliwych odpowiedzi to zawsze wybierze tą dokładną.